js_functions/task2.js

```diff
@@ -0,0 +1,11 @@
+function adultAge (age) {
```

Comment:
function name should represent the action, use more informative name like `getIsAdult`